lib/Service/Social/DiasporaProvider.php

```diff
@@ -30,31 +30,57 @@ class DiasporaProvider implements ISocialProvider {
 	/** @var IClientService */
 	private $httpClient;
 
-	/** @var boolean */
+	/** @var bool */
 	private $looping;
 
+	/** @var string */
+	public $name = "diaspora";
```

Comment:
did not debug, but the diaspora provider seems to be broken. tried with `podmin@joindiaspora.com `, which works on master.

Reply:
after adding unit tests and getting them to work, i can now confirm that the username you provided results in a successful profile image download